ui/app/registration/controllers/createPatientController.js

```diff
@@ -51,9 +51,30 @@ angular.module('bahmni.registration')
                     }).value();
                 };
 
-                _.chain(personAttributeTypes)
+                var isDateType = function (personAttributeType) {
+                    return personAttributeType.format === "org.openmrs.util.AttributableDate";
+                };
+
+                var isDefaultValueToday = function (personAttributeType) {
+                    if (defaults[personAttributeType.name].toUpperCase() === "today".toUpperCase()) {
```

Comment:
@nehagupta2123, Please change it to use `.toLowerCase` as you are hardcoding **today** (in lowercase)